produced using:
awk -vq="'" '{printf("    %s%s%s,\n", q, $0, q)}' PLD-doc/license-tags.txt

--- rpmlint-0.71/TagsCheck.py~	2006-03-01 00:00:07.000000000 +0200
+++ rpmlint-0.71/TagsCheck.py	2006-03-01 23:51:36.000000000 +0200
@@ -106,50 +106,697 @@
     'X11/Window Managers/Tools',
     )
 
-# liste grabbed from www.opensource.org/licenses
+# see http://cvs.pld-linux.org/PLD-doc/license-tags.txt
 
 DEFAULT_VALID_LICENSES = (
-    'GPL',
-    'LGPL',
-    'GFDL',
-    'OPL',
+    'Academic Free License v2.0',
+    'Activision/Raven, see Documentation',
+    'AFL v2.0 or GPL v2',
+    'AFL v2.1 or GPL v2',
+    'AFPL',
+    'Aladdin FPL and partially LGPL',
+    'Aladdin Free Public Licence (see LICENCE)',
+    'Aladdin Free Public License',
+    'Alladin Free Public License (see Public.htm)',
+    'almost free, distributable',
+    'AMS (BSD like; look at COPYRIGHT)',
+    'Apache',
+    'Apache 1.1',
+    'Apache/BSD-like',
+    'Apache/BSD Style',
+    'Apache Group',
+    'Apache Group License',
+    'Apache license',
+    'Apache License',
+    'Apache License 2.0',
+    'Apache License style',
+    'Apache-like',
+    'Apache Software License',
+    'Apache Software License 1.1',
+    'Apache Software License 2.0',
+    'Apache Software License v1.1',
+    'Apache Software License v2',
+    'Apache Software License, Version 2.0',
+    'Apache-style License',
+    'Apache v1.1',
+    'Apache v2.0',
+    'Apache/W3C',
+    'Apache/W3C/Public Domain',
+    'Apple Free Copyright',
+    'APSL',
+    'APSL / Other (see COPYING)',
     'Artistic',
+    'Artistic 2.0',
+    'Artistic and GPL',
+    'Artistic for code, GPL v2 for icons',
+    'Artistic (generally free)',
+    'Artistic, GPL or BSD',
+    'Artistic (see LICENSE.html)',
+    'as-is',
+    'ASIS',
+    'ASL',
+    'ASL v2.0',
+    'ATI',
+    'ATI Binary (parts are GPL)',
+    'AT&T Open Source',
+    'avas is public domain, ace is on Statlib',
+    'Beer-ware, distributable',
+    'BeOpen Python Open Source License',
+    'BitKeeper',
+    'BitTorrent Open Source License',
+    'Boost Software License and others',
+    'Borland, some modifications are BSD-like licensed (generally free)',
     'BSD',
+    'BSD almost without source',
+    'BSD, Artistic or GPL',
+    'BSD/GPL',
+    'BSD, LGPL',
+    'BSD (library), GPL (Linux kernel module)',
+    'BSD License',
+    'BSD-like',
+    'BSD-like/Apache',
+    'BSD-like (Apple Computer, Inc.)',
+    'BSD-like/Apple Public Source License v2.0 (see COPYING)',
+    'BSD-like (Bell Communications Research)',
+    'BSD-like (except for festival.el, which is on GPL)',
+    'BSD-like, GPL',
+    'BSD-like or GPL',
+    'BSD-like (or GPLv2 for core - see LICENSE)',
+    'BSD-like/Public Domain (see source)',
+    'BSD-like + restricted vendor's name usage (see copyright file)',
+    'BSD-like (see COPYING)',
+    'BSD-like (see COPYRIGHT)',
+    'BSD-like (see docs)',
+    'BSD-like (see included license.txt)',
+    'BSD-like (see LEGAL_NOTICE)',
+    'BSD-like (see LICENSE)',
+    'BSD-like (see LICENSE.txt)',
+    'BSD-like (see README.txt)',
+    'BSD-like with partial source only',
+    'BSD-like, with restrictions',
+    'BSD-like w/o adv. clause',
+    'BSD & NRL',
+    'BSD or Artistic or GPL',
+    'BSD (see LICENCE)',
+    'BSD (see LICENSE_BINARY)',
+    'BSD style',
+    'BSD-style',
+    'BSD w/o adv. clause',
+    '(C) 1997, 1998 Norman Walsh (Free)',
+    '(c) 1999 APC, inc.',
+    'Cambridge University Technical Services Limited',
+    'Caucho Developer Source License',
+    'CCA v2.0',
+    'CDDL v1.0',
+    'Charityware',
+    'CHECK FIRST!',
+    'check first if it's GPL',
+    'check first if it's really GPL',
+    'Check /usr/share/doc/cedega/copyright',
+    'Check with djb@koobera.math.uic.edu',
+    'chntpw',
+    'CNRI',
+    'CNRI OPEN SOURCE LICENSE',
+    'commercial',
+    'Commercial',
+    'commercial (Copyright (C) RealNetworks, Inc.)',
+    'commercial (Copyright (C) RealNetworks, Inc.)',
+    'Commercial (Full 5 user licence for free - registration may be required)',
+    'Commercial, redistributable (see LICENSE)',
+    'Commercial (see EULA)',
+    'commercial with 30-day trial period (see LICENSE)',
+    'Common Public License v0.5',
+    'Copyright 1995-2002 (MIT) (INRIA), (L)GPL compatible',
+    'Copyright (C) 1995,1996,1997,1998,1999,2000,2001 Sebastian Rahtz <s.rahtz@elsevier.co.uk>',
+    'Copyright (C) 1995,1996,1997,1998 Sebastian Rahtz <s.rahtz@elsevier.co.uk>',
+    'Copyright (c) 2000-2001, eGenix.com Software GmbH, All Rights Reserved (partly only for non-comercial use)',
+    'Copyright (C) by Bellcore, non-profit use and redistribution permitted (see COPYING for details)',
+    'Copyright (C) OMRON Corporation, OMRON Software Co., Ltd. (see README)',
+    'Copyright (C) OMRON Corporation, OMRON Software Co., Ltd. (see README)',
+    'Copyright (distributable)',
+    'Copyrighted by Murat Balaban (distributable, see COPYING)',
+    'Copyrighted, freely distributable if unmodified',
+    'CPL',
+    'CPL 1.0',
+    'CPL/GPL',
+    'CPL/LGPL',
+    'CPL or GPL or LGPL',
+    'CPL v0.5',
+    'CPL v1.0',
+    'CPL (yiterm), GPL (the rest)',
+    'Crawl GPL',
+    'Creative Commons and/or Public Domain',
+    'Creative Commons License (see COPYING)',
+    'Creative Commons License (see legalcode.html)',
+    'custom',
+    'Custom',
+    'custom, distributable',
+    'Custom (do, what you want)',
+    'custom (EULA), non-distributable',
+    'custom (free)',
+    'custom (free, Artistic-like)',
+    'Custom Licence by (c) 2003-2004 Linuxant inc. All rights reserved.',
+    'custom, non-distributable',
+    'custom OPL (see License),FDL',
+    'custom (see LICENSE* files)',
+    '(C) Windmill Software 1983',
+    'Dell',
+    'Demo of commercial application',
+    'Design Science',
+    'Design Science License (DSL)',
+    'Different per subpackage',
+    'Digital's paranoid but open-source license',
+    'distributable',
+    'Distributable',
+    'distributable (BSD, GPL, LGPL, Artistic; see COPYING)',
+    'distributable - Copyright 1989,1990 by Paul Sutcliffe Jr.',
+    'distributable (Copyright (C) 2000 Kevin Sullivan)',
+    'Distributable for PLD',
+    'Distributable for PLD until 31 Dec 2006 - http://distribute.opera.com/ (otherwise restricted, see file LICENSE)',
+    'Distributable, free for demo-, evaluation- and personal use',
+    'distributable if unchanged and without charge',
+    'distributable if unchanged (see LICENSE)',
+    'distributable if unmodified',
+    'distributable (mostly GPL)',
+    'distributable - most of it GPL',
+    'distributable not for profit, free use',
+    'distributable provided copyright and permission notices are included',
+    'distributable (see COPYING)',
+    'distributable (see COPYING for details)',
+    'distributable (see COPYING.txt)',
+    'distributable, see included LICENSE',
+    'distributable (see LICENSE)',
+    'distributable (see LICENSE for details)',
+    'distributable (see README)',
+    'distributable (see README.gz)',
+    'distributable (see readme.txt)',
+    'Distributable with author's permission',
+    'distributable with free-unices distros, free for non-profit non-commercial purposes',
+    'distributable without modifications',
+    'distribution restricted (http://www.adobe.com/products/acrobat/distribute.html)',
+    'DivX Open License',
+    'DJB (free to use, see http://cr.yp.to/distributors.html)',
+    'DJB (http://cr.yp.to/distributors.html)',
+    'DJB http://cr.yp.to/distributors.html',
+    'DJB (http://cr.yp.to/qmail/dist.html)',
+    'DOOM3',
+    'dual licensed using BSD-Style and LGPL',
+    'dunno',
+    'Eclipse Public License',
+    '- (enter Apache/GPL/LGPL/BSD/BSD-like/other license name here)',
+    '- (enter GPL/GPL v2/LGPL/BSD/BSD-like/other license name here)',
+    '(enter GPL/LGPL/BSD/BSD-like/Artistic/other license name here)',
+    '- (enter GPL/LGPL/BSD/BSD-like/other license name here)',
+    'EPL v1.0',
+    'Eric Bischoff, Mark Galassi, Jochem Huhmann, Steve Cheng, and Frederik Fouvry; GPL 2.0',
+    'extended GPL v2',
+    'FDL',
+    'for use with KSocrat only',
+    'free',
+    'Free',
+    'free/commercial (Gentleware AG)',
+    'Free copying + BSD license',
+    'Free (Copyright (C) 1999 The OpenJade group)',
+    'Free (Copyright (C) Technische Universitaet Berlin)',
+    'Free for educational use, non-distributable',
+    'Free for non comercial use',
+    'Free for non-comercial use, non-distributable',
+    'free for non-commercial',
+    'Free for non-commercial distribution',
+    'free for non-commercial purposes',
+    'Free for non-commercial purposes (see README and data sets help pages for details)',
+    'free for non-commercial, scientific use only in HDF software',
+    'Free for non-commercial use',
+    'Free for noncommercial use',
+    'Free for non-commerical use',
+    'free for non-profit (non-distributable)',
+    'Free for personal or research use, distributable',
+    'Free for personal use',
+    'free for use in HDF software',
+    'Free + GPL (for some parts of code)',
+    'Free (libs), (c) Microsoft Corporation (headers)',
+    'freely distributable',
+    'Freely distributable',
+    'Freely Distributable',
+    'freely distributable for non-commercial use, GPL (mpglib)',
+    'freely distributable (SGI)',
+    'freely distributable & usable (JServ), LGPL (JSDK)',
+    'Freely redistributable and some GPL',
+    'free, not-distributable',
+    'free of charge for non-profit or internal use',
+    'Free, only Copyright must be preserved',
+    'free (see cd.html)',
+    'free (see LICENSE)',
+    'free (see manuals)',
+    'free (see README)',
+    'Free (see README)',
+    'Free Software',
+    'free to use',
+    'Free to use',
+    'Free to use but restricted',
+    'Free to use, but see http://www.cogsci.princeton.edu/~wn/',
+    'Free to use, non-distributable',
+    'free to use, no restrictions',
+    'Free to use (see COPYING)',
+    'free use, but modifications must be notified to the author',
+    'free use, distributable',
+    'Free use, no source',
+    'free use on Linux (see LICENSE)',
+    'freeware',
+    'Freeware',
+    'Freeware (academic non commercial use)',
+    'Freeware for non-commercial use, not distributable without source',
+    'Freeware (non-commercial use and distribution only)',
+    'Freeware with exceptions - see docs',
+    'GFDL',
+    'Giftware',
+    'GNU',
+    'GNU v2',
+    'GPL',
+    'GPL 2',
+    'GPL and Artistic',
+    'GPL and FIPL (see the license-fi.txt)',
+    'GPL and LGPL',
+    'GPL (application), BSD-like (libnetbrake)',
+    'GPL/Artistic',
+    'GPL (Atari800), distributable if unmodified (xf25 with ROMs)',
+    'GPL/BSD',
+    'GPL/BSD (partial source)',
+    'GPL (but see description)',
+    'GPL, but see description for restrictions',
+    'GPL, but SIFT algorithm may require license in some countries',
+    'GPL, but source URL and version must be specified',
+    'GPL, but uses commercial xaudio library',
+    'GPL/Creative Common ShareAlike License',
+    'GPL (except for images - see COPYING)',
+    'GPL except music',
+    'GPL except .pak file',
+    'GPL (except ROMs)',
+    'GPL (except TOS image)',
+    'GPL (eyes), BSD (tips)',
+    'GPL (forced only by gzip code, basically BSD-like)',
+    'GPL for code, artwork and sounds can be redistributed only with Maelstrom',
+    'GPL (for code only)',
+    'GPL (for personal and educational use; non-distributable in binary form)',
+    'GPL, free or free for non-commercial use',
+    'GPL/Freeware',
+    'GPL / GPL v2',
+    'GPL (interface) / closed source (actual driver)',
+    'GPL (Lemmings), unknown (the rest)',
+    'GPL, LGPL',
+    'GPL/LGPL',
+    'GPL/LGPL/MIT',
+    'GPL-like',
+    'GPL-like/"reserved" (see documentation for details)',
+    'GPL/MIT',
+    'GPL + MySQL FLOSS Exception',
+    'GPL (not Firmware)',
+    'GPL or Artistic',
+    'GPL or Artistic or BSD',
+    'GPL or Blitz artistic license',
+    'GPL or BSD',
+    'GPL or commercial',
+    'GPL or custom (see COPYING)',
+    'GPL or FTL',
+    'GPL or MIT or Free',
+    'GPL or Netscape Public License 1.1',
+    'GPL or QPL',
+    'GPL, partially free (see COPYRIGHT.vmo2wav)',
+    'GPL, partially LGPL',
+    'GPL, partialy LGPL',
+    'GPL, perhaps except for doom3.wad',
+    'GPL (perl code also Artistic)',
+    'GPL/PHP',
+    'GPL plus linking exception',
+    'GPL (program only)',
+    'GPL/QPL',
+    'GPL QPL BSD Eiffel Artistic Aladdin',
+    'GPL script with binaries under unknown license',
+    'GPL (see COPYING)',
+    'GPL (unknown)',
+    'GPL (unofficially)',
+    'GPL (utilities), BSD (library)',
+    'GPL v1',
+    'GPL v1+',
+    'GPL v1+ Artistic',
+    'GPL v1+ (except Carp.pm) or Artistic',
+    'GPL v1+ (except C-Cookbook) or Artistic',
+    'GPL v1+ oe Artistic',
+    'GPL v1+ or Artisric',
+    'GPL v1 or Artistic',
+    'GPL v1+ or Artistic',
+    'GPL v1+ or Artistic (C library also LGPL)',
+    'GPL v1+ or Artistic / distributable (pTK library)',
+    'GPL v1+ or Artistic except commercial distribution on CD-ROM etc.',
+    'GPL v1+ or Artistic or Apache Software License 1.1',
+    'GPL v1+ or Artistic or (C library only) LGPL',
+    'GPL v1+ or Artistic, parts BSD-like',
+    'GPL v1+ or Artistic (parts on ICU License)',
+    'GPL v1+ or Artistic (with one restriction, see LICENSE)',
+    'GPL v1+ or or Artistic',
+    'GPL v2',
+    'GPLv2',
+    'GPL v2',
+    'GPL v 2',
+    'GPL v.2',
+    'GPL v2+',
+    'GPL v2.1',
+    'GPL v2 and LGPL',
+    'GPL v2/Artistic',
+    'GPL v.2/BSD/LGPL',
+    'GPL v2 (CHECK IT FIRST, could be ASL)',
+    'GPL v2 (except for images - see COPYING)',
+    'GPL v2, FDL',
+    'GPL v2 for main code, BSD-like for fec.c',
+    'GPL v2, GFDL (docs)',
+    'GPL v.2/LGPL',
+    'GPL v2 / LGPL',
+    'GPL v2+ / LGPL',
+    'GPL v2+/LGPL v2+',
+    'GPL v2 / LGPL v2.1',
+    'GPL v2, LGPL v2.1',
+    'GPL v2+/LGPL v2.1+',
+    'GPL v2/LGPL v2.1 (see COPYING)',
+    'GPL v2+ + MySQL FLOSS Exception',
+    'GPL v2 or Artistic',
+    'GPL v2+ or Artistic',
+    'GPL v2 or later',
+    'GPL v2+ or MPL v1.1',
+    'GPL v2 or see LICENCE',
+    'GPL v2 + others - see README for details',
+    'GPL v2+, portions on Intel's BSD-like license (see README.*)',
+    'GPL v2. See COPYRIGHT.svm.cpp for the copyright of the svm C++ code.',
+    'GPL v2+ (see LICENSE)',
+    'GPL v2+, with explicit permission to link against Qt',
+    'GPL v2+ with free software exception',
+    'GPL v2+ with linking exception',
+    'GPLv2 with OpenSSL exception',
+    'GPL v2 with OpenSSL linking exception',
+    'GPL v2+ with unlimited link permission',
+    'GPL v2+ (yp6) / MIME++ Software License Agreement',
+    'GPL with BSD-licensed, closed-source plugins',
+    'GPL with clause allowing linking to AFPL Ghostscript',
+    'GPL (with exception clauses) and EAPL',
+    'GPL with limited linking exception',
+    'GPL with non-commercial additions',
+    'GPL with unlimited link permission',
+    'GPL (w/o sources - sic! - so non-distributable, possibly illegal)',
+    'GPL/XFree86',
+    'GP v1+ or Artistic',
+    'gSOAP public license',
+    'HCA',
+    'hp',
+    'HPL 1.0',
+    'http://cr.yp.to/distributors.html (free to use)',
+    'http://www.demarc.org/license/ (Free for non-commercial use)',
+    'IBM Binary Code License',
+    'IBM Common Public License v1.0',
+    'IBM or GPL',
+    'IBM or GPL (code), Artistic or GPL (examples and docs)',
+    'IBM proprietary, distributable (see license.txt)',
+    'IBM Public License',
+    'IBM Public License v1.0',
+    'IBM Public License Version 1.0',
+    'IBM Public License Version 1.0 - Jikes Compiler, http://ibm.com/research/jikes/license/license3.htm',
+    'IBPP License (based on Mozilla Public License)',
+    'improve-ware',
+    'Industrial Light & Magic',
+    'Interbase Public License 1.0',
+    'IPL',
+    'It's free, but requires ads of mks.com.pl on your WWW (see licencja.txt)',
+    'Kohnian',
+    'Kupu',
+    'LaTeX Project Public License',
+    'LaTeX Project Public License (http://www.latex-project.org/lppl.txt)',
+    'LGPL',
+    'LGPL 2.1',
+    'LGPL 2.1+',
+    'LGPL 2.1+ (except sm library <free to use> and sm_idle <PSF>)',
+    'LGPL but see LICENSE',
+    'LGPL (communication libraries), CPL (yiterm), GPL (the rest)',
+    'LGPL / Free (dtree)',
+    'LGPL/GPL',
+    'LGPL/GPL/Proprietary',
+    'LGPL (KarmaLib), GPL (modules)',
+    'LGPL (library), GPL (application)',
+    'LGPL (library), GPL (gmdb2)',
+    'LGPL (library), GPL (ipmicmd)',
+    'LGPL (library), GPL (programs)',
+    'LGPL (library), GPL (utilities)',
+    'LGPL (library), GPL (utils)',
+    'LGPL-like',
+    'LGPL/MPL/MIT X11',
+    'LGPL + OCaml linking exception',
+    'LGPL or Artistic',
+    'LGPL or BSD',
+    'LGPL or GPL v1+ or Artistic',
+    'LGPL or MPL',
+    'LGPL (runtime), GPL (tools)',
+    'LGPL v2',
+    'LGPL v2+',
+    'LGPL v2.1',
+    'LGPL v2.1+',
+    'LGPL v2.1/GPL v2',
+    'LGPL v2.1 (libhandle), GPL v2 (the rest)',
+    'LGPL v2.1+ (libjack), GPL v2+ (the rest)',
+    'LGPL v2.1+ (library), GPL v2+ (tools)',
+    'LGPL v2.1 (library), GPL v2 (utilities)',
+    'LGPL v2.1+ or GPL v2+ or Apache v2',
+    'LGPL v2.1+ or GPL v2+ or Apache v2.0+',
+    'LGPL v2.1 or MPL v1.1',
+    'LGPL v2.1+ with OpenSSL linking exception',
+    'LGPL v2/GPL v2',
+    'LGPL v2+/GPL v2+',
+    'LGPL v2+ (library), GPL v2 (utilities)',
+    'LGPL v2+ (library), GPL v2+ (utilities)',
+    'LGPL v2+ or GP2PS License v2+ (see COPYING.GL2PS)',
+    'LGPL v2+ (see COPYING for exceptions)',
+    'LGPL v2+ with the exceptions: http://www.sptk.net/index.php?act=license',
+    'LGPL (wcslib), non-commercial use and distribution (see AST_CONDITIONS)',
+    'LGPL with additional linking exception',
+    'LGPL with amendments (see COPYING)',
+    'LGPL with exceptions (see COPYING.ZZIP)',
+    'LGPL with special exception',
+    'LGPL w/ linking exceptions',
+    'libFoundation license',
+    'Libstk Library License (relaxed LGPL)',
+    'limited/closed source (it will be made available soon under GPL)',
+    'Look into documentation',
+    'Mahogany Artistic or GPL',
+    'MeepZor Consulting Public Licence (MCPL)',
+    'Microsoft EULA (for non-commercial use)',
     'MIT',
-    'QPL',
+    'MIT (base library), GPL (GNU portions in examples)',
+    'MIT (core), SGI (GLU,libGLw) and others - see COPYRIGHT file',
+    'MIT, freely distributable',
+    'MIT+LGPL',
+    'MIT-licensed',
+    'MIT-like',
+    'MIT-like (see LICENSE)',
+    'MIT or LGPL',
+    'MIT, partially Public Domain (see LICENSE)',
+    'MIT/X',
+    'MIT/X11',
+    'MIT/X Consortium/BSD (see README)',
+    'MIT/X Consortium License',
+    'MIT/X (evdev), GPL v2+ (evtest)',
+    'MIT/X + LBNL BSD-style + Zope Public License',
+    'Mixed (BSD and GPL)',
+    'Mixed (distributable), mostly GPL',
+    'Mixed (GPL, LGPL, distributable)',
+    'Mixed (some are GPL, some LGPL, some BSD, but all opensource)',
+    'mixture of UM and Sun licenses',
+    'Modified BSD License',
+    'Modified BSD (see included LICENSE)',
+    'mostly Lucent Public License, Version 1.02',
+    'Mostly Public Domain with Free & GPL additions',
+    'Mozilla Public License',
+    'Mozilla Public License, some parts on other license (distributable)',
     'MPL',
-    'IBM Public License',
-    'Apache License',
+    'MPL 1.0',
+    'MPL 1.1',
+    'MPL 1.1/GPL 2.0/LGPL 2.1',
+    'MPL/GPL',
+    'MPL/LGPL',
+    'MPL/LGPL/GPL',
+    'MPL or GPL',
+    'MPL or MIT/X',
+    'MPL v1',
+    'MPL v1.1 or GPL',
+    'MPL v1.1 or GPL v2 or LGPL v2.1',
+    'MPL v1.1 or GPL v2+ or LGPL v2.1+',
+    'MPL v1.1 (original code) and other licenses (included libraries)',
+    'Nearly BSD, but changed sources must be marked',
+    'Nethack GPL',
+    'New BSD',
+    'non-commercial',
+    'non-commercial distribution and encoding',
+    'non-commercial distribution and use',
+    'non-commercial, for research only',
+    'non-commercially distributable if complete, unmodified and not bundled on CD etc.',
+    'Non-commercial, non-military purposes, w/ and only w/ the voice and language databases available on http://tcts.fpms.ac.be/synthesis/',
+    'non-commercial (see COPYING)',
+    'non-commercial (see license* files)',
+    'non-commercial, see readme.txt',
+    'non-commercial use',
+    'non-commercial use and distribution',
+    'non-commercial use and distribution (see EMS_CONDITIONS)',
+    'non-commercial use and distribution (see GSD_CONDITIONS)',
+    'non-commercial use and distribution (see MERS_CONDITIONS)',
+    'non-commercial use and distribution (see NDF_CONDITIONS)',
+    'non-commercial use and distribution (see PAR_CONDITIONS)',
+    'non-commercial use and distribution (see PCS_CONDITIONS)',
+    'non-commercial use, freely distributable',
+    'non distributable',
+    'non-distributable',
+    'Nondistributable but free (See Bitstream-Cyberfonts-licence.txt)',
+    'non-distributable, free use',
+    'non-distributable, no source',
+    'nondistributable on term of Internet Explorer 4 EULA',
+    'non-distributable (see license.html and http://java.sun.com/docs/redist.html)',
+    'non distributable (see license.txt)',
+    'non-profit',
+    'non-profit - see README',
+    'no restrictions',
+    'not precise, free for single users',
+    'Not specified',
+    'NPL 1.1',
+    'NRL/TIN',
+    'nVidia',
+    'nVidia Binary',
+    'NVIDIA proprietary, distributable if unmodified (see LICENSE)',
+    'OpenContent',
+    'Open Group Public License',
+    'OpenLDAP Public License',
+    'Open Publication License',
+    'Open Publication License v1.0+',
+    'OpenSceneGraph Public Licence (based on LGPL with exceptions)',
+    'Open Software License',
+    'Open Software License v2.0',
+    'Open Source',
+    'Open Source, mostly BSD like, some LGPL/GPL',
+    'Open source (MPICH), BSD-like (MPI-2-C++)',
+    'Open Source (see LICENSE file for details)',
+    'OpenSSL (Apache-style)',
+    'OPL',
+    'OSI Approved',
+    'OSL 1.0 (http://www.opensource.org/licenses/osl.php)',
+    'OSL v1.1',
+    'Other License(s)',
+    'Other License(s), see package',
+    'own, incompatibile with GPL',
+    'partially GPL',
+    'PB EULA',
+    'Philips B.V.',
+    'PHP',
+    'PHP 2.0',
+    'PHP 2.02',
+    'PHP 3.0',
+    'PHP 3.00',
+    'PHP/BSD',
     'PHP License',
-    'Public Domain',
-    'Modified CNRI Open Source License',
-    'zlib License',
-    'CVW License',
-    'Ricoh Source Code Public License',
-    'Python license',
-    'Vovida Software License',
-    'Sun Internet Standards Source License',
-    'Intel Open Source License',
-    'Jabber Open Source License',
-    'Nokia Open Source License',
-    'Sleepycat License',
-    'Nethack General Public License',
-    'Common Public License',
-    'Apple Public Source License',
-    'X.Net License',
-    'Sun Public License',
-    'Eiffel Forum License',
-    'W3C License',
-    'Zope Public License',
-    'Lucent Public License',
-    'Design Science License',
-    'LaTeX Project Public License',
-    'CECILL',
-    # non open source licences:
+    'PHP License 3.0',
+    'Portable Batch System (PBS) Software License',
+    'Postcardware',
+    'probably free, but no source',
     'Proprietary',
-    'Freeware',
+    'Proprietary (Free for non-business use. Busines use requires registration.)',
+    'Proprietary (See copyright)',
+    'Proprietary, use is permited. Copyright (C) 2002, AVM GmbH. All rights reserved.',
+    'PSF',
+    'Public Domain',
+    'Public Domain and LGPL',
+    'Public Domain, BSD, GPL (see COPYING.txt)',
+    'Public Domain, partially BSD-like',
+    'Public Domain (see Legal)',
+    'Public Domain (see README)',
+    'Public Use License v1.0',
+    'Python',
+    'Python (MIT style)',
+    'Python Software Foundation License',
+    'Python-style',
+    'Q3A EULA, PB EULA',
+    'Qmail',
+    'QPL',
+    'redistributable',
+    'redistributable for non-commercial use',
+    'redistributable (see LICENSE)',
+    'relaxed BSD and (L)GPL-compatible',
+    'relaxed LGPL (libraries), and Public Domain (docs)',
+    'restricted, non-distributable',
+    'restricted, non-distributable (DIVXNETWORKS EULA)',
+    'restricted, non-distributable (see COPYRIGHT)',
+    'restricted, non-distributable (see COPYRIGHT.html)',
+    'restricted, non-distributable (Sun Binary Code License - see LICENSE.txt)',
+    'restricted, non-distributable (Sun Binary Code License - see URL)',
+    'restricted, non-distributable (Sun Community Source License - see URL)',
+    'restricted, no source',
+    'Restricted, see file LICENSE',
+    'restricted (see LICENSE.txt)',
+    'RPSL or GPL v2+',
+    'RTCW-ETEULA',
+    'RTCW EULA, PB EULA',
+    'Ruby',
+    'Ruby-alike',
+    'Ruby License',
+    'Ruby's',
+    'SCSL',
+    'see LICENSE',
+    'see LICENSE file (redistribution restricted)',
+    'see license.terms',
+    'several free software licenses',
+    'SGI Free Software License B v1.1',
+    'SGI MIT-like',
     'Shareware',
-    'Charityware'
+    'SHORTEN SOFTWARE LICENSE',
+    'SIL Open Font License',
+    'sleepycat',
+    'Sleepycat public license (GPL-like, see LICENSE)',
+    'Special (see Copyright Notice)',
+    'Spin Public license',
+    'SRP Open Source',
+    'SugarCRM Public License',
+    'Sun Binary Code License',
+    'Sun Binary Code License (restricted, non-distributable)',
+    'Sun Public License or LGPL',
+    'Thai Open Source Software Center Ltd (distributable)',
+    '"THE BEER-WARE LICENSE" (Revision 42)',
+    'The Clarified Artistic License',
+    'The Glasgow Haskell Compiler License',
+    'The program itself BSD, the man page and icon GPL',
+    'The Ruby License',
+    'The VIGRA Artistic License',
+    'Unlimited distribution.',
+    'Unlimited use and distribution (see LICENCE).',
+    'Very Restrictive',
+    'vim',
+    'VistaLab Public License v1.5 (GPL v2+ with exception, see COPYING)',
+    'Vovida Software License',
+    'W3C',
+    'W3C Copyright (BSD like)',
+    'W3C / PHP 2.0',
+    'W3C (see: http://www.w3.org/Consortium/Legal/copyright-software.html)',
+    'wxWidgets Licence (LGPL with exception)',
+    'wxWindows Library v. 3 (LGPL derivative)',
+    'wxWindows License',
+    'X11',
+    'X11/MIT',
+    'X Consortium-like',
+    'X Consortium (MIT-like)',
+    'XFree86 1.1',
+    'X License',
+    'XML-RPC C Library License',
+    'Yahoo! DomainKeys Public License Agreement v1.1',
+    'Zend License, distributable only if unmodified and for free (see LICENSE)',
+    'Zend Studio License',
+    'ZLIB',
+    'zlib/libpng',
+    'zlib/libpng license',
+    'Zope Public License (ZPL)',
+    'Zope Public License (ZPL), GPL',
+    'ZPL',
+    'ZPL 2.0',
+    'ZPL 2.1',
     )
 
 DEFAULT_PACKAGER = 'PLD bug tracking system ( http://bugs.pld-linux.org/ )'
